x-pack/plugins/event_log/server/es/context.ts

```diff
@@ -23,6 +23,7 @@ export interface EsContext {
   waitTillReady(): Promise<boolean>;
   readonly initialized: boolean;
   readonly retryDelay: number;
+  shouldSetExistingAssetsToHidden: boolean;
```

Comment:
I'm open to a better name, I was debating between `isServerless` or something more descriptive like `shouldSetExistingAssetsToHidden`.